### ISSUE TITLE
Issue Highlight Focus Indicator Can Have Insufficient Contrast

### ISSUE THREAD

**Issue by MajoMaldonado** [equalizedigital/accessibility-checker#1768]
Location / URL: https://edac-vpat.instawp.co/water-department-consumer-report-now-available/?edac=12986
 
Screenshot: 

<img width="411" height="165" alt="Image" src="https://github.com/user-attachments/assets/af4f44a6-7909-42c3-8fde-535ca0122d96" />

WCAG S.C: 1.4.11 Non-text Contrast (Level AA)

Issue Description: When the Accessibility Checker highlights an element associated with an issue, the visual focus indicator does not provide sufficient contrast against the surrounding page content. As a result, the highlighted element is difficult to distinguish from adjacent colors, making it challenging for users to visually identify the element currently being emphasized.

**Comment by amberhinds** [comment]
This one is going to be difficult because we have no idea what the background color will be. I think it would be worth switching to the browser default for focus?

**Comment by pattonwebz** [comment]
The pink-purple colour was picked because it passes against white and black bacgounds with at least a 4.5:1 ratio. I doubt think be we can use browser focus styles because the focus isn't actually on the element that's highlighted, it stays in the frontend highlighter widget.

This one may need to just be mentioned in the VPAT as known to have instances it is not ideal.

**Comment by amberhinds** [comment]
Oh, I didn't think this was about the pink color. I thought this was about the light blue indicator on the active button. @MajoMaldonado can you please clarify?

I agree the pink should not be changed.

**Comment by amberhinds** [comment]
@pattonwebz, it just occurred to me that we could add a setting option to let people change this color. Then if, say they have a hot pink website, they could choose a different color for highlighting that they know will contrast with their color scheme.

**Comment by MajoMaldonado** [comment]
@pattonwebz @amberhinds Yes, this makes reference to the pink color that highlights the elements.

**Comment by amberhinds** [comment]
Thanks. My initial comment about using the browser focus style is incorrect, then, as I initially misunderstood. I think we can leave this issue open as a setting would be ideal for non-technical users, but I am changing this to a low priority.

@MajoMaldonado, I created a new doc that tells people how they can fix this: https://equalizedigital.com/accessibility-checker/how-to-change-the-frontend-highlight-outline-color/

In the ACR, Non-Text Contrast should be supports with a note:
In some themes, the front end highlight color may not sufficiently contrast with the element's background color. If needed, site administrators can [change the highlight outline color with CSS.](https://equalizedigital.com/accessibility-checker/how-to-change-the-frontend-highlight-outline-color/)
